client/src/app/admin/panel/settings/admin-panel-email-settings.js

```diff
@@ -196,22 +192,47 @@ class AdminPanelEmailSettings extends React.Component {
                 <Form {...this.getFormProps()}>
                     <div className="row">
                         <div className="col-md-7">
-                            <FormField label={i18n('SUBJECT')} name="subject" validation="TITLE" required
-                                       fieldProps={{size: 'large'}} />
+                            <FormField
+                                fieldProps={{size: 'large'}}
+                                label={i18n('SUBJECT')}
+                                name="subject"
+                                validation="TITLE"
+                                required />
                         </div>
                     </div>
 
-                    <FormField key="text1" label={i18n('TEXT') + '1'} name="text1" validation="TEXT_AREA" required
-                               decorator={'textarea'}
-                               fieldProps={{className: 'admin-panel-email-settings__text-area'}} />
-                    {(form.text2 || form.text2 === "") ?
-                        <FormField key="text2" label={i18n('TEXT') + '2'} name="text2" validation="TEXT_AREA" required
-                                   decorator={'textarea'}
-                                   fieldProps={{className: 'admin-panel-email-settings__text-area'}} /> : null}
-                    {(form.text3 || form.text3 === "") ?
-                        <FormField key="text3" label={i18n('TEXT') + '3'} name="text3" validation="TEXT_AREA" required
-                                   decorator={'textarea'}
-                                   fieldProps={{className: 'admin-panel-email-settings__text-area'}} /> : null}
+                    <FormField
+                        fieldProps={{className: 'admin-panel-email-settings__text-area'}}
+                        label={i18n('TEXT') + '1'}
+                        key="text1"
+                        name="text1"
+                        validation="TEXT_AREA"
+                        required
+                        decorator={'textarea'} />
+                    {
+                        (form.text2 || form.text2 === "") ?
+                            <FormField
+                                fieldProps={{className: 'admin-panel-email-settings__text-area'}}
+                                label={i18n('TEXT') + '2'}
+                                key="text2"
+                                name="text2"
+                                validation="TEXT_AREA"
+                                required
+                                decorator={'textarea'} /> :
+                            null
+                    }
+                    {
+                        ((form.text3 || form.text3 === "") && (selectedIndex !== 1 && selectedIndex !== 2)) ?
```

Comment:
using `selectedIndex` can be a bit dangerous, because order can change. Can we use something like a list of template codes (like "RECOVER_PASSWORD") and filter by inclusion?